File: src/main/java/eu/happybe/openapi/mysql/query/CreatePartyQuery.java
package eu.happybe.openapi.mysql.query;

import eu.happybe.openapi.mysql.AsyncQuery;

import java.sql.SQLException;
import java.sql.Statement;

public class CreatePartyQuery extends AsyncQuery {

    public String owner;
    public String server;

    public CreatePartyQuery(String owner, String server) {
        this.owner = owner;
        this.server = server;
    }

    @Override
    public void query(Statement statement) throws SQLException {

    }
}

File: src/main/java/eu/happybe/openapi/mysql/query/CreatePartyQuery.kt
package eu.happybe.openapi.mysql.query

import eu.happybe.openapi.mysql.AsyncQuery
import java.sql.SQLException
import java.sql.Statement

class CreatePartyQuery(var owner: String, var server: String) : AsyncQuery() {
    @Throws(SQLException::class)
    override fun query(statement: Statement) {
    }
}